### PR TITLE
MES-2161 Extract additional information presentational component to handle form repopulation (MES-1129)

### PR DESCRIPTION
**Pull request by johnmorleybjss**
## Description and relevant Jira numbers
* Extract additional information capture into a dedicated presentational component
* Have `AdditionalInformationComponent` handle it's own form control and repopulation of the form value
* Emit an event back to the Office page when additional information changes to update the store
* Move `ShowMeQuestionComponent` into the office `components` package

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

### CODE REVIEW

File: src/pages/office/components/additional-information/additional-information.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export class AdditionalInformationComponent implements OnChanges {
+
+  @Input()
+  additionalInformation: string;
+
+  @Input()
+  formGroup: FormGroup;
+
+  @Output()
+  additionalInformationChange = new EventEmitter<string>();
```
Comment (ammarhaiderbjss) [inline src/pages/office/components/additional-information/additional-information.ts]: Did you have a go at passing a dispatcher down from the parent?

Comment (johnmorleybjss) [reply src/pages/office/components/additional-information/additional-information.ts]: Not yet, will have a try for this PR, see how it compares.

Comment (johnmorleybjss) [reply src/pages/office/components/additional-information/additional-information.ts]: As discussed, going with the `EventEmitter` for now.

Comment (johnmorleybjss) [reply src/pages/office/components/additional-information/additional-information.ts]: As discussed after comparison with #331, going with `EventEmitter` for now.

**Pull request merged**